qa/rpc-tests/test_framework/mininode.py

```diff
@@ -192,13 +193,13 @@ def deser_string_vector(f):
 def ser_string_vector(l):
     r = ""
     if len(l) < 253:
-        r = chr(len(l))
+        r = struct.pack("B", len(l))
     elif len(l) < 0x10000:
-        r = chr(253) + struct.pack("<H", len(l))
+        r = struct.pack("B", 253) + struct.pack("<H", len(l))
     elif len(l) < 0x100000000L:
-        r = chr(254) + struct.pack("<I", len(l))
+        r = struct.pack("B", 254) + struct.pack("<I", len(l))
     else:
-        r = chr(255) + struct.pack("<Q", len(l))
+        r = struct.pack("B", 255) + struct.pack("<Q", len(l))
```

Comment:
Thanks, yes I agree with regard to keeping the minimum diff, but indeed it's better to do this small cleanup (which someone is bound to do later) at the same time.